twitter/urls.py

```diff
@@ -23,4 +23,5 @@
     path('admin/', admin.site.urls),
     path('', home, name='home'),
     path('user/', include('user.urls')),
+    path('tweet/', include('tweet.urls')),
```

Comment:
Read about restful 
- https://restfulapi.net/resource-naming/
- https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5

Reply:
Never use CRUD function names in URIs
![image](https://github.com/GaganDureja/twitter/assets/53633318/ee1b18e0-92ec-4e2f-ae58-4d9027f4f8e6)

I did this 
```python  
  path('registrations/create', views.createRegistration, name='createRegistration'),  
  path('sessions/create', views.createSession, name='createSession'),
```

Reply:
Yes, thats why i names them sessions and registerations.

Reply:
As mentioned, change the Path to `tweets/....`